pages/osx/cot.md

```diff
@@ -0,0 +1,24 @@
+# cot
+
+> The Plain-Text Editor for macOS.
+> More information: <https://coteditor.com/>.
+
+- Start CotEditor:
+
+`cot`
+
+- Open a new blank document:
+
+`cot --new`
+
+- Open specific files:
+
+`cot {{path/to/file1 path/to/file2 ...}}`
+
+- Open and wait until finished editing a specific file:
+
+`cot --wait {{path/to/file}}`
+
+- Open a file with the cursor at a specific line and column:
```

Comment:
```suggestion
- Open a specific file with the cursor at a specific line and column:
```

Reply:
Same deal here. I can make a new PR to give `mate` the same wording as `cot` once this PR is merged.